spark-atlas-connector/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

```diff
@@ -0,0 +1,487 @@
+/*
```

Comment:
This is copied from Spark: https://github.com/apache/spark/blob/master/external/kafka-0-10-sql/src/test/scala/org/apache/spark/sql/kafka010/KafkaTestUtils.scala

I expect Kafka also provides some mechanism to run embedded cluster or another lightweight approaches, but the license will not be matter so my 2 cents it seems OK.

Reply:
Ah I realized the class can be pulled with test jar... Will remove this class.